cuml/README.md

```diff
@@ -0,0 +1,104 @@
+# cuML Notebooks
+
+## Summary
+
+* `dbscan_demo`: Notebook showcasing density-based spatial clustering of applications with noise (dbscan) algorithm comparison between cuML and scikit-learn.
```

Comment:
"`*_demo`: First_word" --> should be "`*_demo`: first_word" to match the style currently in place.